kvbc/test/CMakeLists.txt

```diff
@@ -153,18 +153,6 @@ target_link_libraries(kv_blockchain_db_editor_test PUBLIC
 )
 endif(BUILD_ROCKSDB_STORAGE)
 
-if(USE_S3_OBJECT_STORE)
```

Comment:
Why was this test removed?

Reply:
This test claimed to test a storage factory, but essentially it didn't